new_twitter.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./new_twitter.css" />
+</head>
+<body>
+    <div class = 'container'>
```

Comment:
Don't add white space around the property/value pairs. It works, but it goes against convention - I don't know any project that uses whitespace in this way. Also, be sure to use double quotes instead of single quotes

`<div class="container">`